doc/releases/release-notes-3.7.rst

```diff
@@ -72,6 +72,17 @@ Drivers and Sensors
 
 * CAN
 
+  * Added optional ``min-bitrate`` devicetree property for CAN transceivers.
+  * Added devicetree macros :c:macro:`DT_CAN_TRANSCEIVER_MIN_BITRATE` and
+    :c:macro:`DT_INST_CAN_TRANSCEIVER_MIN_BITRATE` for getting the minimum supported bitrate of a CAN
+    transceiver.
+  * Added support for specifying the minimum bitrate supported by a CAN controller in
+    :c:macro:`CAN_DT_DRIVER_CONFIG_GET` and :c:macro:`CAN_DT_DRIVER_CONFIG_INST_GET`.
```

Comment:
The link in the documentation is broken for these. Most likely because they are for internal drivers only?

Reply:
Ah, right. I'v reworded the entry slightly and removed the linking attempt. Thanks.